server/pulp/server/managers/status.py

```diff
@@ -55,6 +58,7 @@ def get_broker_conn_status():
         conn.connect()
         conn.release()
         return {'connected': True}
-    except:
+    except Exception, e:
+        _logger.exception(e)
```

Comment:
Re-reading these helped me out: 
https://docs.python.org/2/library/logging.html#logging.Logger.debug
https://docs.python.org/3/library/logging.html#logging.Logger.debug

In py3, there's a line reading "If an exception tuple (in the format returned by sys.exc_info()) _or an exception instance is provided_" (emphasis mine). In py2, that sentence reads line is "If an exception tuple (in the format returned by sys.exc_info()) is provided"; the "or an exception instance" bit is not there.